src/clients.rs

```diff
@@ -149,9 +149,13 @@ pub async fn create_http_clients(
             let port = service_config.port.unwrap_or(default_port);
             let mut base_url = Url::parse(&service_config.hostname).unwrap();
             base_url.set_port(Some(port)).unwrap();
+            let request_timeout = match service_config.timeout {
```

Comment:
Instead of this, should we set the default via default trait ? that way we can just do `service_config.timeout` without match everywhere ?